packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -137,6 +141,7 @@ export const Control = ({
 
   // Map overlays always have initial dates, so DateRangePicker always has dates on initialisation,
   // and uses those rather than calculating it's own defaults
+  console.log(mapOverlay, defaultDates);
```

Comment:
The last commit on this is called `WIP` and this is still here, shall I hold off review? Perhaps you could delete from the reviews channel and re-post once it's ready?

Reply:
Ah sorry, I just planned on dealing with that one case there as it showed up due to changes in the latest merge from dev, but felt the rest of the code was ready for review? Can hold off if you'd prefer til it's all finished though 👍

Reply:
Oh I should have read right through your PR description, no that's all good I can review as is :-)